src/commands/export-sql.js

```diff
@@ -299,25 +301,31 @@ export class ExportSQLCommand {
 	 */
 	async run() {
 		console.log( `Fetching the latest backup for ${ this.app.name }` );
-		const { latestBackup } = await fetchLatestBackupAndJobStatus( this.app.id, this.env.id );
+		const { latestBackup, jobs } = await fetchLatestBackupAndJobStatus( this.app.id, this.env.id );
```

Comment:
We're calling this function again at line 216, through the `getExportJob` call at line 313, and I don't see a good reason to call this API twice.

Maybe we could try refactor this? We could check if `this.runningJobs` is already populated (and if anyone wants to cache invalidate it, they could nullify `this.runningJobs`), or they could put a parameter whether or not to get a fresh or cached value.

What do you think?

Reply:
Actually this is intentional. getExportJob gives us the job and the job-status. We are interested in refreshing the job status on every steps.